cluster/azure/aks-gitops/variables.tf

```diff
@@ -102,11 +102,6 @@ variable "kubeconfig_filename" {
   default     = "bedrock_kube_config"
 }
 
-variable "kubeconfig_recreate" {
```

Comment:
With the removal of this, if you do an update to the cluster does the config get recreated?

Reply:
No, the user would need to delete the file and run `terraform apply` again to recreate the file with updated credentials which is similar to the current behavior. Currently, user changes the value `kubeconfig_recreate`and run `terraform apply` to recreate the file.

Reply:
Then this will require documenting.

Reply:
Good idea. Added a note on how to recreate/re-download the file.